### PR TITLE
EA Forum Dialogues

### PR DESCRIPTION
**Pull request by oetherington**
Enable dialogues on the EA forum, with a few styling changes. [Figma](https://www.figma.com/file/aMHp60bV7NyYXfsl9VOVO2/Dialogues?type=design&node-id=0-1&mode=design&t=zW7SDMTp5t8NH9eR-0). [Doc](https://docs.google.com/document/d/15z1CRNTiyMFmy_N3JVEs3gLr3se5cpPFt9r7aiHNR1M/edit).

Screenshot 2023-11-20 at 20 22 36

Screenshot 2023-11-20 at 20 22 55

Screenshot 2023-11-20 at 20 23 05

Screenshot 2023-11-20 at 20 23 15

Screenshot 2023-11-20 at 20 23 29

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1205945438850899) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/posts/dialogues/DialogueSubmit.tsx [hunk 1]
```diff
@@ -106,79 +88,60 @@ const DialogueSubmit = ({
...
 
   const requireConfirmation = isLW && collectionName === 'Posts' && !!document.debate;
+  const showShortformButton = !!userShortformId && !isFriendlyUI;
 
   const onSubmitClick = requireConfirmation ? submitWithConfirmation : submitWithoutConfirmation;
 
-
+  const {Row} = Components;
   return (
-    <React.Fragment>
```
Comment (oetherington) [inline packages/lesswrong/components/posts/dialogues/DialogueSubmit.tsx:114]: This component was very messy with a ton of unused stuff, plus it tried to add class-component-esque `PropTypes` even though this is a functional component which confused the typechecker so I removed those too. The only material change in this file is removing the "post to shortform" button.

**Comment by oetherington** [comment]
FYI @agnesstenlund

**Review by jpaddison3 — APPROVED** [review]
LGTM

**Comment by oetherington** [comment]
@agnesstenlund Thanks! I've just pushed commits which should fix all the issues you mentioned. I'll consider this ready to merge tomorrow morning unless you notice anything else (or we can just make follow-up PRs).

**Comment by agnesstenlund** [comment]
woow 🥲 👏

**Comment by agnesstenlund** [comment]
- [ ] only found one issue now, the "writing" box now has the right font size, but the editing boxes are gone (I think they were helpful in making it look like a draft + frame the submit button + distinguishing different people)

Screenshot 2023-11-21 at 09 33 33

- [ ] Also the submit button looks good in dark mode but a bit weird in light mode (has black text instead of white). Seeing this I'd actually also make it grey with black text instead of blue with white, so it doesn't look too much like a big primary button (thinking it could almost be confused with the "published" button). Fine to keep the blue though, just a small thing

**Comment by oetherington** [comment]
@agnesstenlund Thanks! Changes made 😄

**Pull request merged**